cfme/networks/topology.py

```diff
@@ -0,0 +1,200 @@
+import re
+import time
+
+from navmazing import NavigateToAttribute
+
+from cfme.networks.topology_view import TopologyView
+from cfme.utils.appliance import Navigatable
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from wait_for import wait_for
+
+
+class Topology(Navigatable):
+    LEGENDS = '//kubernetes-topology-icon'
+    ELEMENTS = '//kubernetes-topology-graph//*[name()="g"]'
+    LINES = '//kubernetes-topology-graph//*[name()="lines_obj"]'
+
+    def __init__(self, appliance):
+        Navigatable.__init__(self, appliance=appliance)
+        self.legends_obj = []
+        self.elements_obj = []
+        self.lines_obj = []
+        self.element_ref = None
+        self.display_names = None
+        self.view = None
+
+        self.reload_view()
+        self.refresh()
+        self.reload_elements_and_lines()
+        self.reload_legends()
+
+    def __repr__(self):
+        return "<Topology {}>".format(", ".join(self._legends_obj))
+
+    def reload_view(self):
+        self.view = navigate_to(self, 'All')
```

Comment:
This method doesn't make sense for me.  First of all it doesn't reload the view, but navigate to somewhere. If you are already there nothing will happen.